RNFlashList.podspec

```
@@ -9,14 +9,19 @@ Pod::Spec.new do |s|
   s.homepage         = package['homepage']
   s.license          = package['license']
   s.author           = package['author']
-  s.platforms        = { :ios => '11.0', :tvos => '12.0' }
+  s.platforms        = { ios: '12.4', tvos: '12.0' }
   s.source           = { git: 'https://github.com/shopify/flash-list.git', tag: "v#{s.version}" }
   s.source_files     = 'ios/Sources/**/*'
   s.requires_arc     = true
   s.swift_version    = '5.0'
 
-  # Dependencies
-  s.dependency 'React-Core'
+  # Use install_modules_dependencies helper to install the dependencies if React Native version >=0.71.0.
```

@naqvitalha The `RCT_NEW_ARCH_ENABLED` isn't actually set anywhere.

```suggestion
  if fabric_enabled
    s.pod_target_xcconfig = {
      'OTHER_SWIFT_FLAGS' => '-D RCT_NEW_ARCH_ENABLED',
    }
  end

  # Use install_modules_dependencies helper to install the dependencies if React Native version >=0.71.0.
```

And then it would also be required to check if the flag is set at the top level (cannot add suggestion there 😞):
```ruby
fabric_enabled = ENV['RCT_NEW_ARCH_ENABLED']
```